src/blueapi/service/subprocess_handler.py

```diff
@@ -0,0 +1,144 @@
+from multiprocessing import Pool
+import os
+from typing import List, Optional
+from blueapi.config import ApplicationConfig
+from blueapi.core.bluesky_types import Plan
+from blueapi.service.facade import BlueskyHandler
+from blueapi.service.handler import get_handler, setup_handler, teardown_handler
+from blueapi.service.model import DeviceModel, PlanModel, WorkerTask
+from blueapi.worker.event import WorkerState
+from blueapi.worker.task import RunPlan
+from blueapi.worker.worker import TrackableTask
+
+
+class SubprocessHandler(BlueskyHandler):
+    _config: ApplicationConfig
+    _subprocess: Pool
+
+    def __init__(
+        self,
+        config: Optional[ApplicationConfig] = None,
+    ) -> None:
+        self._config = config or ApplicationConfig()
+        self._subprocess = None
+
+    def start(self):
+        if self._subprocess is not None:
+            raise ValueError("Subprocess already running")
+        self._subprocess = Pool(processes=1)
+        self._subprocess.apply(setup_handler, [self._config])
+
+    def stop(self):
+        self._subprocess.apply(teardown_handler)
+        self._subprocess.terminate()
+        self._subprocess = None
```

Comment:
These should be idempotent: If it's started then `start()` should do nothing, ditto for stop